src/content/docs/blog/2025/07/11/Development update: os-tools/index.md

```diff
@@ -0,0 +1,184 @@
+
+The **TL;DR** is that for:
+
+- **Moss:** Existing PRs reviewed, refined and merged (including a PR enabling faster package installation via parallel blitting). The code received various bug fixes and refactors for correctness and maintainability.
+- **Boulder:** Similarly, existing PRs reviewed, refined and merged including a few smaller features and packaging macros being added. Since Boulder uses the moss crate, it now builds packages slightly quicker due to faster buildroot creation. The odd bugfix was also made.
+- **User Experience:** Improved error reporting features were added to both Moss and Boulder to improve the troubleshooting experience for users and packagers. More to be done on this front.
+
+We will be continuing our os-tools work over the coming months with a specific focus on tidying up the code, improving documentation, and ensuring better error handling and status reporting throughout the codebase. This will come in particularly handy when we do the feature work to make us able to produce JSON output as the final part of the alpha2 os-tools milestone.
+
+Overall, this work will help users when they come across unexpected errors and also be beneficial when onboarding new developers.
+
+The JSON output feature, in contrast, is largely targeted at convenient machine parsing of structured output for automation and integration purposes, which we are banking on will come in handy for future development work currently in the planning stages.
+
+
+## Introducing NomadicCore
+
+Before going any further, you may have noticed my name as one of the authors of our mid-year update blog post. If you hang around our Matrix rooms, you will likely already know me but I thought it prudent to provide a formal introduction.
+
+I first became aware of SerpentOS about three years ago but only joined the Matrix chat rooms in September 2023. I'm not actually a developer or have any coding experience, however, I am interested in open source projects and Linux distributions that can help me get the most out of my hardware. I liked what I saw with SerpentOS and over the course of 2024, started getting involved, trying to help out where I could.
+
+Earlier this year, I ended up formally joining the team, around the time of the AerynOS rebrand, taking more of a support / communications role and providing feedback from an "average user" perspective of what I think might be important.
+
+My focus will mainly be around working on our documentation, writing blog posts and engaging on our various social media platforms and Matrix rooms. I'm looking forward to getting stuck in and helping support a Linux distribution I want to use on my various devices.
+
+
+## A deeper dive into our os-tools work
+
+Since their port to Rust, our os-tools have been working well enough. However, we are self aware enough to know that our initial porting efforts left room for improvement, both in code quality and in performance.
```

Comment:
```suggestion
Since their port to Rust, our os-tools have been working well enough. However, we are self-aware enough to know that our initial porting efforts left room for improvement, both in code quality and performance.
```